clients/AlphabotClientLibrary/AlphabotClientLibrary/AlphabotClientLibrary.Shared/Models/WiFiConnectionData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Remove unnecessary using directives.
```suggestion
using System.Net;
```